dali/test/dali_test_single_op.h

```diff
@@ -158,31 +145,29 @@ template<typename ImgType>
 class DALISingleOpTest : public DALITest {
  public:
   inline void SetUp() override {
+    const char *subDir[] = {"", "/png"};
     DALITest::SetUp();
     c_ = (IsColor(ImageType()) ? 3 : 1);
-    jpegs_.clear();
+    imageDecrs_[t_jpegImgType].clear();
 
     const auto flags = GetImageLoadingFlags();
 
-    if (flags & t_loadJPEGs) {
-      LoadJPEGS(images::jpeg_test_images, &jpegs_);
-      if (flags & t_decodeJPEGs)
-        DecodeImages(DALI_RGB, jpegs_, &jpeg_decoded_, &jpeg_dims_);
-    }
-
-    if (flags & t_loadPNGs) {
-      LoadImages(images::png_test_images, &png_);
+    for (int j = t_undefinedImgType; ++j < kLastImgType;) {
```

Comment:
for (int j = t_undefinedImgType + 1; j < kLastImgType; j++) would be a bit easier to understand probably.

Reply:
DONE